docs/section15/Rule15-8.md

```diff
@@ -0,0 +1,26 @@
+# Transformers - Rule 15-8
+**Rule ID:** Rule 15-8  
+**Rule Description:** Transformer efficiency reported in Proposed RMR equals User RMR  
+**Rule Assertion:** User RMR = Proposed RMR  
+**Appendix G Section:** Transformers  
+**Appendix G Section Reference:**  Table G3.1 15. Distribution Transformers  
+**Data Lookup:** Table 8.4.4  
+**Evaluation Context:**  Each Data Element   
+**Applicability Checks:**
+1. Transformer is DRY_TYPE
+2. U_RMR transformer capacity is within range of Table 8.4.4  
+
```

Comment:
Another applicability check is that tranformer efficiency exceeds minimum in Table 8.4.4.